src/ncp/ncp_base_mtd.cpp

```diff
@@ -1010,6 +1010,38 @@ template <> otError NcpBase::HandlePropertyGet<SPINEL_PROP_THREAD_PENDING_DATASE
     return EncodeOperationalDataset(dataset);
 }
 
+#if OPENTHREAD_ENABLE_JOINER
```

Comment:
Since this feature can be enabled/disabled on NCP/OT, how about we add a `SPINEL_CAP_THREAD_JOINER` in `spinel.h` and emit it from `NcpBase::HandlePropertyGet<SPINEL_PROP_CAPS>`? (so we can know in wpantund if this feature is present/enabled).